simpleflow/execute.py

```diff
@@ -409,7 +411,25 @@ def main():
         metavar='N',
         help='error file descriptor',
     )
+    parser.add_argument(
+        '--kill-children',
+        action='store_const',
+        const=True,
+        default=False,
+        help='kill child processes on exit',
+    )
     cmd_arguments = parser.parse_args()
+
+    def kill_child_processes():
+        process = psutil.Process(os.getpid())
+        children = process.children(recursive=True)
+
+        for child in children:
```

Comment:
I think it's okay since children list come from https://pythonhosted.org/psutil/#psutil.Process.children call.
BTW: I just saw that and I think I'll use `wait_procs` instead of sleep it: https://pythonhosted.org/psutil/#psutil.wait_procs